tests/periph_pwm/main.c

```diff
@@ -68,12 +167,94 @@ int main(void)
         }
 
         state += step;
-        if (state <= 0 || state >= (int)STEPS) {
+        if (state <= 0 || state >= (int)OSC_STEPS) {
             step = -step;
         }
 
-        xtimer_periodic_wakeup(&last_wakeup, INTERVAL);
+        xtimer_periodic_wakeup(&last_wakeup, OSC_INTERVAL);
     }
 
     return 0;
 }
+
+int power(int argc, char** argv){
+    if (argc != 3) {
+        printf("usage: %s <dev> <state>\n", argv[0]);
+        printf("\tdev: device by number between 0 and %d\n", PWM_NUMOF-1);
+        printf("\tstate:\n");
+        printf("\t\t0: power off\n");
+        printf("\t\t1: power on\n");
+        return 1;
+    }
+
+    errno = 0;
+    uint32_t dev = strtol(argv[1], NULL, 10);
```

Comment:
here and in the rest of the file: any reason for parsing 'long int' (strtol) vs simply parsing 'int' (atoi)? Since the target range is fairly small, it seems to me that
```c
unsigned dev = (unsigned)atoi(argv[1]);
```
would be the more efficient choice...